.circleci/local_publish_helpers.sh

```diff
@@ -270,19 +270,12 @@ function setAwsAccountCredentials {
 function runE2eTest {
     FAILED_TEST_REGEX_FILE="./amplify-e2e-reports/amplify-e2e-failed-test.txt"
 
-    if [ -z "$FIRST_RUN" ] || [ "$FIRST_RUN" == "true" ]; then
-        startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
-        setNpmRegistryUrlToLocal
-        changeNpmGlobalPath
-        cd $(pwd)/packages/amplify-e2e-tests
-    fi
```

Comment:
This doesn't work on windows and has already been pulled up to circleci yaml for linux (and thus is redundant there).